src/parser/user-extractor-transformer.ts

```diff
@@ -0,0 +1,68 @@
+import configuration from "../configuration/config-reader";
+import { GetActivity } from "../get-activity";
+import { GitHubIssue, GitHubIssueComment } from "../github-types";
+import { Result, Transformer } from "./processor";
+
+/**
+ * Creates entries for each bounty hunter with its associated comments.
+ */
+export class UserExtractorTransformer implements Transformer {
+  private readonly _configuration = configuration["user-extractor"];
+
+  get enabled(): boolean {
+    return true;
+  }
+
+  /**
+   * Checks if the comment is made by a human user, and if it not a command.
+   * @param comment
+   */
+  _checkEntryValidity(comment: GitHubIssueComment) {
+    return !!comment.user && !!comment.body && comment.user.type === "User";
+  }
+
+  _extractBountyPrice(issue: GitHubIssue) {
+    if (this._configuration["redeem-bounty"] === false) {
```

Comment:
The team had a whole debate on how the keys from the config should be camel cased in order to directly map to identifier names (like variables) @rndquu rfc

Reply:
We decided to use camel cased parameters ([example](https://github.com/ubiquity/ubiquibot-config/blob/6f102946f0b511cbb7a53221d9230841fdc61642/.github/ubiquibot-config.yml)) for the bot's config on partners' side. I guess the same convention should be applied to plugin configs (if `redeem-bounty` refers to it).